src/main/webapp/src/app/login-signup-page/login-signup.component.ts

```diff
@@ -81,13 +119,15 @@ export class LoginComponent {
    *
    * @param data object holding user data that will be used as a param in the post request
    */
-  postData(data: any): void {
+  postData(data: User): void {
     const httpOptions = {
       headers: new HttpHeaders({
         'Content-Type': 'application/json',
       }),
       params: new HttpParams().set('userData', JSON.stringify(data)),
     };
-    this.httpClient.post<any>('/user', null, httpOptions);
+    this.httpClient
+      .post<any>('/user', null, httpOptions)
+      .subscribe(result => {});
```

Comment:
why are we subscribing here? do we not care about if the request was successful or an error?

Reply:
Its blank because I originally had a console log there and took it out but in the next pr I will use the response to display a conformation or error message to the user about their account being successfully created.